Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -184,10 +185,19 @@ - (void)configureAppearance {
     UIColor *rangeTitleColor = [self colorForCurrentStateInDictionary:self.appearance.titleColors];
 
     if (self.titleLabel.text) {
+        if (self.wholeMonthSelection) {
+            self.titleLabel.attributedText = [BPKFont attributedStringWithFontStyle:BPKFontStyleTextHeading5
```

Comment:
Could we move the FontStyle (BPKFontHeading5) to a constant? (So it can be easily re-used and changed when design changes?